### DISCUSSION TITLE
April 18th 2024 Community Call

### DISCUSSION THREAD

**Discussion by StevenBucher98** [PowerShell/PowerShell#21461] [Community Call]
9:30am PST start

Agenda:
- PSSummit Recap and experience share
    - Organizers: James Petty and Missy Januszko
    - Attendee: Matthew Gills
    - Onramp: Aspen Clark
    - MSFT Team: @StevenBucher98 
- DSC v3 update @SteveL-MSFT
- ACR Support @alerickson  
- Docs Update @sdwheeler

**Comment by StartAutomating** [comment]
I might be able to show off some Docker / PowerShell / Rocker fun.

During summit, I dropped a new module [Rocker](https//github.com/StartAutomating/Rocker).  It intercepts calls to docker to give a better PowerShell experience (object output, tab completion, pipelined parameters, and strongly typed parameter support).

Since Summit, I've been adding Docker support to a number of modules.  It would be great to show/tell and talk about where this could go.

**Comment by therealgill** [comment]
3D PowerShell logo, finally posted! 

https://www.thingiverse.com/thing:6584540
